packages/uni_app/lib/controller/local_storage/database/app_restaurant_database.dart

```diff
@@ -14,15 +14,17 @@ class RestaurantDatabase extends AppDatabase<List<Restaurant>> {
           CREATE TABLE RESTAURANTS(
           id INTEGER PRIMARY KEY,
           ref TEXT,
-          name TEXT)
+          namePt TEXT
+					namePt TEXT)
           ''',
             '''
           CREATE TABLE MEALS(
           id INTEGER PRIMARY KEY AUTOINCREMENT,
           day TEXT,
           type TEXT,
           date TEXT,
-          name TEXT,
+          namePt TEXT,
+          nameEn TEXT,
```

Comment:
Changes to the database creation script require a version bump! Let me know if you need help on how to do that!

Reply:
Nevermind, I got it, the database is not being used, okay!